cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskManager.java

```diff
@@ -132,6 +177,20 @@ public void addBalancingProposals(Collection<BalancingProposal> proposals,
       if (!_inProgressPartMovementsByBrokerId.containsKey(p.destinationBrokerId())) {
         _inProgressPartMovementsByBrokerId.put(p.destinationBrokerId(), 0);
       }
+
+      // Increase pending task sensors.
+      BalancingAction balancingAction = p.balancingAction();
+      if (balancingAction == BalancingAction.REPLICA_MOVEMENT) {
+        _executionCounterByName.get(COUNTER_REPLICA_MOVE_PENDING).inc();
```

Comment:
Hmm, not sure if this is the best way of exposing the sensor values. We are using the dropwizardMetrics counter to do the counting. This could be inconsistent with the actual number of the proposals in the data structure. It seems better to avoid using the sensors counter as the program counter in this case.